css/header.css

```diff
@@ -1,34 +1,91 @@
 @import url("/css/default.css");
 
+.container {
+  width: 80%;
+  height: max-content;
+  margin: 0 auto;
+}
+
 header {
+  padding: 2.5rem 1rem;
+  width: 100%;
   display: flex;
   justify-content: space-between;
-  padding: 2rem;
+  align-items: center;
 }
 
-.header-brand-title {
+.header-left {
+  font-size: 1.5rem;
+  font-weight: 900;
+}
+
+.header-left a {
   color: var(--primary-color);
-  font-size: 1.6rem;
-  font-weight: 500;
-  cursor: pointer;
+  text-decoration: none;
+}
+
+.small-text {
+  font-size: 0.7rem;
 }
 
 .header-right {
   display: flex;
+  justify-content: space-evenly;
   align-items: center;
 }
 
-.header-right .header-link {
-  font-size: 1rem;
+.header-right li {
+  margin: 0 1rem;
+  list-style: none;
 }
 
 .header-icon {
-  margin-left: 1.2rem;
   font-size: 1.8rem;
 }
 
-@media only screen and (max-width: 600px) {
-  .header-right .header-link {
-    display: none;
+.header-right a {
+  font-weight: 800;
+  color: rgb(90, 90, 90);
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 0.9rem;
+  transition: all 0.3s ease;
+}
+
+.header-right a:hover {
+  color: black;
+}
+
+@media screen and (max-width: 1200px) {
+  .container {
+    width: 90%;
+  }
+}
+
+@media screen and (max-width: 800px) {
+  header {
+    padding: 2.5rem 1rem;
+  }
+
+  .header-left {
+    font-size: 1.1rem;
+  }
+
+  .header-right li {
+    margin: 0 0.5rem;
+  }
+
+  .header-right li a {
+    font-size: 0.8rem;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .container {
```

Comment:
I think you need to change order too, add media queries from smaller to bigger viewport size. the opposite may cause some problems later. Idk just lil suggession.

Reply:
Will look into this, thanks